Ports/python3/patches/0002-Tweak-configure.patch

```diff
@@ -0,0 +1,80 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Oskar Skog <oskar@oskog97.com>
+Date: Wed, 27 Sep 2023 15:21:55 +0300
+Subject: [PATCH] Tweak configure
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+Merged patches from Linus Groh, Julian Offenhäuser, Oskar Skog:
+
+As usual, make the `configure` script recognize Serenity. Also set
+`MACHDEP` (which is used for `sys.platform`) to a version-less
+`serenityos`, even when not cross-compiling.
+
+Set name of shared libpython
+
+Compile with CCSHARED=-fPIC
```

Comment:
Please keep proper authorship information. We are mainly basing this on Linus' patch, so we should keep that metadata and just add on all the other authors as coauthors.

The individual changes to libpython and CCSHARED are also minor enough to not warrant a separate heading. If you want to combine it with the existing commit message, feel free to.

```suggestion
From: Linus Groh <mail@linusgroh.de>
Date: Fri, 14 Jan 2022 23:35:57 +0330
Subject: [PATCH] Tweak configure
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

As usual, make the `configure` script recognize Serenity. Also set
`MACHDEP` (which is used for `sys.platform`) to a version-less
`serenityos`, even when not cross-compiling.

Co-Authored-By: Julian Offenhäuser <offenhaeuser@protonmail.com>
Co-Authored-By: Oskar Skog <oskar@oskog97.com>
```